job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext/sloccount.groovy

```diff
@@ -0,0 +1,11 @@
+job('example') {
+    publishers {
+        sloccount {
+            pattern('build/result.xml')
+            encoding('UTF-8')
+            commentIsCode()
+            numBuildsInGraph(2)
```

Comment:
Rename this to `buildsInGraph`. Use the UI labels instead of the XML element names as an orientation for naming things.